app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -343,6 +345,19 @@ public void setExternalIdValidationEnabled(boolean externalIdValidationEnabled)
     public void setMinSupportedAppVersions(Map<String,Integer> map) {
         this.minSupportedAppVersions = (map == null) ? new HashMap<>() : map;
     }
+    
+    /** {@inheritDoc} */
+    @Override
+    public Map<String,String> getPushNotificationARNs() {
+        return pushNotificationARNs;
+    }
```

Comment:
Is it worth adding to DynamoStudyDaoTest.crudStudy() to make sure we can write these values to / read from DDB?